gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -3,13 +3,6 @@
         "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
         "https://checkstyle.org/dtds/configuration_1_3.dtd">
 
-<!--
-    Palantir Baseline Checkstyle configuration.
-    Authors: Robert Fink, Brian Worth, Merrick Zoubeiri, and many other contributors. Based in part on http://checkstyle.sourceforge.net/google_style.html
-    Please keep checks alphabetized with one exception: "relaxed" checks are grouped together at the bottom for easier disabling.
-    Check-specific comments reference documents internal to Palantir and can be safely ignored or removed.
- -->
```

Comment:
Removed this comment because it didn't write well - it ends up looking like:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><!--
    Palantir Baseline Checkstyle configuration.
    Authors: Robert Fink, Brian Worth, Merrick Zoubeiri, and many other contributors. Based in part on http://checkstyle.sourceforge.net/google_style.html
    Please keep checks alphabetized with one exception: "relaxed" checks are grouped together at the bottom for easier disabling.
    Check-specific comments reference documents internal to Palantir and can be safely ignored or removed.
 --><module name="Checker">
```

This comment isn't particularly useful. In particular the checks are no longer alphabetized.

Reply:
I'd prefer not to make this change in the same PR as a bugfix.

Reply:
Are you opposed to the change? I'm happy to make another PR, but if you're just going to approve that one it seems a bit silly.

I probably should have removed this in https://github.com/palantir/gradle-baseline/pull/1603.

Reply:
I don't really have an opinion one way or the other on the comment.